### ISSUE TITLE
Remove /team in SG

### ISSUE THREAD

**Issue by ashleyindigital** [TheNexusMC-LLC/Issues#95]
Ok, so players have the ability to do the command /team. I believe this should be removed because if a moderator joins an sg that someone is reported in then everyone has access to see who is in the spectator list. Sometimes hacker's teammates would type /team and check if a moderator has joined and if one joins then can warn they're friend or same thing for the hacker, they can check /team and toggle off their hacks if they see a moderator join.

**Comment by TheMoltenGuy** [comment]
You can still see whether or not there is a staff member in the game if you do /list and compare the amount of spectators with the amount that the scoreboard on the right says, because Staff who are in incognito will not show up in /list but the spectator total on the right will still increase.

 Though I agree that /team should be removed because you can as a tribute use the command to see the health of other players which is a bit unfair.

**Comment by ashleyindigital** [comment]
Yeah, but newer players on the server who are hacking aren't going to realize the whole side bar thing :p
